src/material-experimental/mdc-chips/chip-grid.ts

```diff
@@ -154,7 +154,7 @@ export class MatChipGrid extends _MatChipGridMixinBase implements AfterContentIn
    * Implemented as part of MatFormFieldControl.
    * @docs-private
    */
-  get empty(): boolean { return this._chipInput.empty && this._chips.length === 0; }
+  get empty(): boolean { return this._chipInput.empty && (!this._chips || this._chips.length === 0); }
```

Comment:
Should we null check for the `_chipInput` too? AFAIK it's not guaranteed to be defined either.

Reply:
Probably should - just wasn't an issue internally. I added a change to include that too